src/main/kotlin/lotto/domain/Rank.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.domain
+
+enum class Rank(
+    val matchCount: Int,
+    val prize: Long
+) {
+    MISS(0, 0),
+    FOURTH(3, 5000),
+    THIRD(4, 50000),
+    SECOND(5, 1500000),
```

Comment:
Kotlin에서 숫자는 언더바( _ ) 로 구분하면 더 가독성있게 표현할 수 있습니다 :)
```suggestion
    MISS(0, 0),
    FOURTH(3, 5_000),
    THIRD(4, 50_000),
    SECOND(5, 1_500_000),
    ...
```